org.caione.GScope.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "org.caione.GScope",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "command": "gscope",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name" : "gtksourceview",
+            "config-opts" : [ "--disable-Werror", "--disable-doc" ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "branch" : "gnome-3-24",
+                    "url" : "https://gitlab.gnome.org/GNOME/gtksourceview.git"
```

Comment:
https://github.com/flathub/flathub/wiki/App-Requirements#stable-releases-reproducible-builds

Reply:
makes sense. Fixing.

Reply:
I'd suggest just using the tarballs release over tags, just smaller and faster downloads.